From 2bc238d199fc67f92683e76af204dbcb547eeb1e Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Wed, 11 Jun 2014 11:53:02 -0400
Subject: [kernel] smp/generic-ipi/locking: Fix misleading smp_call_function_any() description

Message-id: <1402487594-26332-3-git-send-email-prarit@redhat.com>
Patchwork-id: 83640
O-Subject: [RHEL7.1 PATCH BZ 1105192 02/14] generic-ipi/locking: Fix misleading smp_call_function_any() description
Bugzilla: 1105192
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Neil Horman <nhorman@redhat.com>

commit 15e71911fcc655508e02f767a3d9b8b138051d2b
Author: Xie XiuQi <xiexiuqi@huawei.com>
Date:   Mon Jul 29 11:52:24 2013 +0800

    generic-ipi/locking: Fix misleading smp_call_function_any() description

    Fix locking description: after commit 8969a5ede0f9e17da4b9437
    ("generic-ipi: remove kmalloc()"), wait = 0 can be guaranteed
    because we don't kmalloc() anymore.

    Signed-off-by: Xie XiuQi <xiexiuqi@huawei.com>
    Cc: Sheng Yang <sheng@linux.intel.com>
    Cc: Peter Zijlstra <a.p.zijlstra@chello.nl>
    Cc: Jens Axboe <jens.axboe@oracle.com>
    Cc: Rusty Russell <rusty@rustcorp.com.au>
    Link: http://lkml.kernel.org/r/51F5E6F8.1000801@huawei.com
    Signed-off-by: Ingo Molnar <mingo@kernel.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/kernel/smp.c b/kernel/smp.c
index 6d56c4e..948b3ea 100644
--- a/kernel/smp.c
+++ b/kernel/smp.c
@@ -270,8 +270,6 @@ EXPORT_SYMBOL(smp_call_function_single);
  * @wait: If true, wait until function has completed.
  *
  * Returns 0 on success, else a negative status code (if no cpus were online).
- * Note that @wait will be implicitly turned on in case of allocation failures,
- * since we fall back to on-stack allocation.
  *
  * Selection preference:
  * 1) current cpu if in @mask
-- 
1.7.1